package.json

```diff
@@ -68,6 +69,7 @@
     "husky": "^8.0.3",
     "jest": "^29.4.3",
     "lint-staged": "^13.2.0",
+    "openapi-typescript": "^5.4.1",
```

Comment:
I would not make this `dev dependency` upgradable, I would just set it to `5.4.1` bc higher versions don't support Swagger 2.0

Reply:
should be fine. `^` allows just for minor and patch updates.